src/overlays/actors/ovl_Obj_Ghaka/z_obj_ghaka.h

```diff
@@ -5,11 +5,17 @@
 
 struct ObjGhaka;
 
+typedef void (*ObjGhakaActionFunc)(struct ObjGhaka*, GlobalContext*);
+
 typedef struct ObjGhaka {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x28];
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ObjGhakaActionFunc actionFunc;
+    /* 0x160 */ char unk160[0x8];
+    /* 0x168 */ s16 unk_168;
 } ObjGhaka; // size = 0x16C
 
 extern const ActorInit Obj_Ghaka_InitVars;
 
+extern CollisionHeader D_06003CD0;
```

Comment:
Any particular reason this is in the `.h`?

Reply:
When looking at other files that had a CollisionHeader, it was placed in the header so I figured why not put it there too